pkg/entities/data.go

```diff
@@ -9,6 +11,12 @@ type Data struct {
 }
 
 type Payload struct {
-	SensorId int `bson:"sensor_id" json:"sensor_id"`
-	Value    int `bson:"value" json:"value"`
+	SensorId int         `bson:"sensor_id" json:"sensor_id"`
+	Value    interface{} `bson:"value" json:"value"`
+}
+
+// DataPublish represents the incoming publish data command
+type DataPublish struct {
```

Comment:
I think this structure is coupled with the specific event received through `amqp` transport and will be used only to unmarshal this event. It's similar to the network `msgs` we defined in the `babeltower`. Please, move it to a similar file.

Reply:
Done!

Reply:
@esdrasjnr now it's fine! 
I'd just suggest you move the `msg.go` to the `network` directory.

Reply:
Done!